lib/scenic/adapters/postgres.rb

```diff
@@ -166,6 +171,16 @@ def try_index_create(index)
         connection.execute("ROLLBACK TO SAVEPOINT #{index.index_name}")
         "index '#{index.index_name}' on '#{index.object_name}' is no longer valid and has been dropped."
       end
+
+      def version_number
```

Comment:
Great news! Personally I don't think the conditional to use the method if it exists is worth it (I think it might be confusing...), so I'm going to just go with `connection.send(:postgresql_version)` for now.